s/cmacros.ss

```diff
@@ -419,7 +419,7 @@
   rv64nb    trv64nb
   la64le    tla64le
 )
-
+<
```

Comment:
stray character?

Reply:
Yes, thanks. (Because I use M-< and M-> to jump to the start/end of the buffer in Emacs, I'm constantly mistyping and immediately jumping away from the mistake, leaving stray `<`s and `>`s in files. Probably I should learn a new keybinding.)